presto-hive/src/main/java/com/facebook/presto/hive/HiveSessionProperties.java

```diff
@@ -229,6 +230,15 @@ public HiveSessionProperties(HiveClientConfig hiveClientConfig, OrcFileWriterCon
                         "Default storage format for new tables or partitions",
                         hiveClientConfig.getHiveStorageFormat().toString(),
                         false),
+                new PropertyMetadata<>(
+                        COMPRESSION_CODEC,
+                        "Compression codec for new tables or partitions",
```

Comment:
nit: Be consistent with https://prestodb.github.io/docs/current/connector/hive.html#hive-configuration-properties :) 
```
The compression codec to use when writing files
```